Update README.md (#6106)
zeit/next.js: packages/next/README.md

404 or 500 errors are handled both client and server side by a default component `error.js`. If you wish to override it, define a `_error.js` in the pages folder:

> ⚠️ Default `error.js` and your `_error.js` components are only used in production. In development you get an error with call stack to know where the error originated from. ⚠️

Reviewer: ```suggestion
⚠️ The `pages/_error.js` component is only used in production. In development you get an error with call stack to know where the error originated from. ⚠️
```
